src/XML/WriterXML.hs

```diff
@@ -60,22 +60,23 @@ elementsToXML [] _ = ""
 elementsToXML (x:xs) n = elementToXML x n ++ elementsToXML xs n
 
 elementToXML :: Element -> Int -> String
+elementToXML e (-1) = elementToXML e (-2)
 elementToXML (Text str) _ = str
-elementToXML (Bold elem) n = "<bold>" ++ elementToXML elem (n + 1) ++ "</bold>"
-elementToXML (Italic elem) n = "<italic>" ++ elementToXML elem (n + 1)
+elementToXML (Bold elem) n = "<bold>" ++ elementToXML elem (-1) ++ "</bold>"
```

Comment:
Bold can contain other elements, not only text, is this not gonna reset the indentation ?

Reply:
If this is the intended code, can you replace n by _ since you don't seem to use it